api-reference/10 UI Widgets/Errors and Warnings/E1058.md

```diff
@@ -3,7 +3,7 @@ id: ErrorsUIWidgets.E1058
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Occurs in the [Scheduler](/api-reference/10%20UI%20Widgets/dxScheduler '/Documentation/ApiReference/UI_Widgets/dxScheduler/') when the specified [startDayHour](/api-reference/10%20UI%20Widgets/dxScheduler/1%20Configuration/startDayHour.md '/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/#startDayHour') is later than the [endDayHour](/api-reference/10%20UI%20Widgets/dxScheduler/1%20Configuration/endDayHour.md '/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/#endDayHour').
```

Comment:
the specified xx is later? what do you mean later? what does it do later?

Reply:
These settings specify hours, so if the `startDayHour` is, for example, 8, then the `endDayHour` should be later than 8, for instance, 12.